packages/babel-plugin-transform-spread/src/index.js

```diff
@@ -97,7 +97,12 @@ export default declare((api, options) => {
 
         const calleePath = skipTransparentExprWrappers(path.get("callee"));
 
-        if (calleePath.isSuper()) return;
+        if (calleePath.isSuper()) {
+          throw path.buildCodeFrameError(
+            "It's not possible to compile spread arguments in `super()` without compiling classes.\n" +
+              "Please add '@babel/plugin-transform-classes' to your Babel configuration.",
```

Comment:
The classes and spread transform have almost identical compat data:

```json
{
"transform-classes": {
    "chrome": "46",
    "opera": "33",
    "edge": "13",
    "firefox": "45",
    "safari": "10",
    "node": "5",
    "ios": "10",
    "samsung": "5",
    "electron": "0.36"
  },
"transform-spread": {
    "chrome": "46",
    "opera": "33",
    "edge": "13",
    "firefox": "36",
    "safari": "10",
    "node": "5",
    "ios": "10",
    "samsung": "5",
    "electron": "0.36"
  },
}
```
Thus it is very unlikely that users of `@babel/preset-env` will hit this issue: if `transform-spread` is enabled, `transform-classes` is also enabled, unless explicitly disabled by `exclude`.

Leave a note here for reference only.